www/_template/reference/configuration.md

```diff
@@ -124,235 +297,157 @@ module.exports = {
 
-- Set the name of the used function to create JSX elements.
+**Type**: `string`
 
-#### buildOptions.jsxFragment | `string` | Default: `React.Fragment` (or `Fragment` if Preact import is detected)
+**Default**: `web_modules`
 
-- Set the name of the used function to create JSX fragments.
+Rename your web modules directory.
 
-### config.testOptions
+### buildOptions.clean
 
-#### testOptions.files | `string[]` | Default: `["__tests__/**/*", "**/*.@(spec|test).*"]`
+**Type**: `boolean`
 
-- The location of all test files.
-- All matching test files are scanned for installable dependencies during development, but excluded from both scanning and building in your final build.
+**Default**: `true`
 
-### config.proxy
+Set to `false` to prevent Snowpack from deleting the build output folder (`buildOptions.out`) between builds.
 
-If desired, `"proxy"` is where you configure the proxy behavior of your dev server. Define different paths that should be proxied, and where they should be proxied to.
+### buildOptions.metaDir
 
-The short form of a full URL string is enough for general use. For advanced configuration, you can use the object format to set all options supported by [http-proxy](https://github.com/http-party/node-http-proxy).
+**Type**: `string`
 
-This configuration has no effect on the final build.
+**Default**: `__snowpack__`
 
-Example:
+Rename the default directory for Snowpack metadata.
 
-```js
-// snowpack.config.json
-{
-  "proxy": {
-    // Short form:
-    "/api/01": "https://pokeapi.co/api/v2/",
-    // Long form:
-    "/api/02": {
-      on: { proxyReq: (p, req, res) => /* Custom event handlers (JS only) */ },
-      /* Custom http-proxy options */
-    }
-  }
-}
-```
+Snowpack-related metadata includes [HMR](/concepts/hot-module-replacement) and [ENV](/reference/configuration#environment-variables) info.
 
-Options:
 
-- **`"path".on`** | `object` (string: function)
-  - `on` is a special Snowpack property for setting event handler functions on proxy server events. See the section on ["Listening for Proxy Events"](https://github.com/http-party/node-http-proxy#listening-for-proxy-events) for a list of all supported events. You must be using a `snowpack.config.js` JavaScript configuration file to set this.
-- All options supported by [http-proxy](https://github.com/http-party/node-http-proxy).
+### buildOptions.sourceMaps
 
-### config.mount
+**Type**: `boolean`
 
-```
-mount: {
-  [path: string]: string | {url: string, resolve: boolean, static: boolean, staticHtml: boolean}
-}
-```
+**Default**: `false`
 
-The `mount` option tells Snowpack which project directories to build and how to build them. Given the following example configuration, you could expect the following results:
+Generates source maps.
 
-```js
-// Example: Basic "mount" usage
-// snowpack.config.json
-{
-  "mount": {
-    "src": "/dist",
-    "public": "/"
-  }
-}
-```
+**_Experimental:_** Lit Element, Preact, Svelte and React sourcemaps in build and dev.
```

Comment:
what does this line mean?

Reply:
I got it from the PR, which said those are the ones it supports?

Reply:
This isn't really true, unfortunately. Our source map support today is just kind of meh, but it's not framework specific. You could instead just write "Still in progress, you may encounter some issues when using source maps until this support is finalized."